src/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+
+class LeaderBoard extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      persons: []
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://fcctop100.herokuapp.com/api/fccusers/top/recent')
+      .then(results => {
+        return results.json();
+      }).then(data => {
+        let persons = data.map((person, i) => {
+          return (
+            <tr key={i}>
+              <td>{i+1}</td>
+              <td>
+                <a href="https://www.freecodecamp.org/">
```

Comment:
Can take to the user's fcc profile instead of fcc. And can also be opened on a new tab.